### PR TITLE
Jarsenault debify

### PR DESCRIPTION
**Pull request by johnathan-arsenault**

### CODE REVIEW

File: .github/workflows/build-docker-image.yml [hunk 1]
```diff
@@ -14,16 +14,15 @@
...
 env:
   PLATFORMS: linux/amd64,linux/arm64
 
 # when to run workflow
 on:
   # # run on push or pull request events for the master branch
-  push:
-    tags:
-      - "v*.*.*"
+  # push:
```
Comment (wsobel) [inline .github/workflows/build-docker-image.yml:23]: Why did you comment this out?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -13,15 +13,15 @@ name: Build MTConnect C++ Agent
 
 on:
   # Enable automated build once verified
-  pull_request:
```
Comment (wsobel) [inline .github/workflows/build.yml:16]: Why did you remove this?

File: .github/workflows/build-deb.yml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      if: steps.cache.outputs.cache-hit == 'true'
+      continue-on-error: true
+      run: |
+        conan remove mtconnect_agent -c
+
+    - name: Build and Test C++ Agent
+      shell: bash
+      run: |
+        export CTEST_OUTPUT_ON_FAILURE=TRUE
+        conan create . --build=missing -pr conan/profiles/gcc  -o shared=${{ matrix.shared }} -o cpack=True -o cpack_name=dist -o cpack_destination=${{ github.workspace }} --test-folder=
```
Comment (wsobel) [inline .github/workflows/build-deb.yml:73]: This will skip tests

Comment (wsobel) [reply .github/workflows/build-deb.yml:73]: Which is fine since it's test in the CI build. Just update name of step.

**Comment by wsobel** [comment]
This looks great. Thanks a lot for putting it together.

**Comment by wsobel** [comment]
Why did you close?

**Comment by johnathan-arsenault** [comment]
@wsobel Would it make more sense to add this to the current build.yaml and output the deb package as artifacts? If so, I think we might be able to expand the matrix to include specific ubuntu version deb packages.

**Comment by wsobel** [comment]
Thanks for the clarification.

I was thinking of making a Ubuntu package but didn't have time. It would be great to get this working. 

- W
